test-harness/src/test/groovy/com/netflix/conductor/test/integration/SubWorkflowSpec.groovy

```diff
@@ -290,6 +294,9 @@ class SubWorkflowSpec extends AbstractSpecification {
             reasonForIncompletion == terminateReason
         }
 
+        and:
```

Comment:
Was this a side-effect? Looking at the terminateWorkflow codeblock, the parent workflow termination should happen synchronously (this needs to be changed as per the new behavior). Given the current behavior, sweep need need be invoked.

Reply:
this behavior has indeed been changed